src/createDispatcher.js

```diff
@@ -34,11 +34,17 @@ const kickstart = { type: '_INIT_' }
 export default function createDispatcher(opts = {}) {
   const dispatcher = new Subject()
 
+  // Question: what is this intended for? I see it used below in
+  // ways that make it seem like the Symbol is for scope hiding
+  // (as they were defined for), not sure what the intention is though
+  // Ah, this gets used to provide hidden annotations on a function the caller provided
```

Comment:
Yes, just for the store caching. This might be problematic in environments that return new functions on each require! Then an encapsulated reducer could help as well, and make caching less necessary. For now though, this is still an issue.